framework/source/class/qx/util/Serializer.js

```diff
@@ -122,7 +122,7 @@ qx.Class.define("qx.util.Serializer",
      * @param dateFormat {qx.util.format.DateFormat?} If a date formater is given,
      *   the format method of this given formater is used to convert date
      *   objects into strings.
-     * @return {String}
+     * @return {Object}
```

Comment:
Actually, it returns a whole bunch of types. It can return `null`, `Array`, `String`, or `Object` depending on what type is being converted.

Reply:
Would then {var} be more adequate?

Reply:
Yes, either that, or `{null|Array|String|Object}`. A bit of text describing when each applies would be great, too.